spark/src/main/java/org/apache/iceberg/spark/ExceptionUtil.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.spark;
+
+import java.io.IOException;
+import org.apache.iceberg.exceptions.NoSuchTableException;
+import org.apache.iceberg.exceptions.RuntimeIOException;
+import org.apache.iceberg.exceptions.ValidationException;
+import org.apache.spark.sql.AnalysisException;
+
+public class ExceptionUtil {
+
+  private ExceptionUtil() {
+  }
+
+  /**
+   * Converts Checked Exceptions to Unchecked Exceptions.
+   *
+   * @param exception the exception object expected to be unchecked.
+   * @return - checked exception.
```

Comment:
I think the javadoc for the arg and return value is the opposite to what is actually done as we converted checked exceptions into unchecked.

Reply:
Also nit: we don't add `-` to Javadoc

Reply:
Fixed this.